spec/cypress/cypress_helper.rb

```diff
@@ -0,0 +1,33 @@
+# This is loaded once before the first command is executed
+
+begin
+ require 'database_cleaner'
+rescue LoadError => e
+ puts e.message
+end
+
+begin
+  require 'factory_bot_rails'
+rescue LoadError => e
+  puts e.message
+  begin
+    require 'factory_girl_rails'
+  rescue LoadError => e
+    puts e.message
+  end
+end
```

Comment:
I think we shouldn't check for `factory_girl_rails`. 

Also, I would just require without begin-rescue blocks, unless we expect this file to run in production environment.

Reply:
Happy to clean this up, but again, this is factory-on-rails added code